### DISCUSSION TITLE
The command 'docker' could not be found in this WSL 2 distro.

### DISCUSSION THREAD

**Discussion by homealim2012** [microsoft/WSL#11339] [General]
在wsl使用docker发生错误，错误信息如下：
<img width="515" alt="微信图片_20240322113221" src="https://github.com/microsoft/WSL/assets/48354217/c1367c14-3353-4891-a027-ea99c1435d09">
在powershell里面显示：
<img width="239" alt="微信图片_202403221132213" src="https://github.com/microsoft/WSL/assets/48354217/a0f0a18e-9f67-48bd-9472-fd775213d8d4">
在docker desktop里面显示：
![微信截图_20240322113411](https://github.com/microsoft/WSL/assets/48354217/1844b4a0-b25c-4a94-8ea5-8b73db4357f2)

请问是什么原因导致的？该如何处理？

**Comment by CrayonL** [comment]
点下refresh试试
